tools/chocolateyinstall.ps1

```diff
@@ -4,21 +4,21 @@ $toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
 
 Get-ChildItem -Path "$toolsDir\tailscale.cer" | Import-Certificate -CertStoreLocation Cert:\LocalMachine\TrustedPublisher
 
-$url = 'https://pkgs.tailscale.com/stable/tailscale-ipn-setup-1.34.1.exe'
-
-
 $packageArgs = @{
-  packageName    = $env:ChocolateyPackageName
-  unzipLocation  = $toolsDir
-  fileType       = 'exe'
-  url            = $url
+  SoftwareName   = 'Tailscale'
+  PackageName    = $env:ChocolateyPackageName
+  UnzipLocation  = $toolsDir
+  FileType       = 'msi'
 
-  softwareName   = 'Tailscale'
+  Url            = 'https://pkgs.tailscale.com/stable/tailscale-setup-1.34.2-x86.msi'
+  Checksum       = '9e9868fed2823f8e16f0f2f858a863d4a4b8ba31e9dfe95d9ca5d556c52858cc'
+  ChecksumType   = 'sha256'
 
-  checksum       = '302b5ef05ad97d39c81624d77882b67adfef2e68da4c9e2a9ebbf5667c29a3b0'
-  checksumType   = 'sha256'
+  Url64Bit       = 'https://pkgs.tailscale.com/stable/tailscale-setup-1.34.2-amd64.msi'
```

Comment:
This is what [Chocolatey says about Arm64](https://docs.chocolatey.org/en-us/create/functions/get-osarchitecturewidth#notes):
> As of 0.10.14+, ARM64 architecture will automatically select 32bit width as there is an emulator for 32 bit and there are no current plans by Microsoft to ship 64 bit x86 emulation for ARM64. For more details, see https://github.com/chocolatey/choco/issues/1800#issuecomment-484293844.

Reply:
What I mean, though, is that you're missing https://pkgs.tailscale.com/stable/tailscale-setup-1.34.2-arm64.msi in this change.

Our NSIS installer includes 3 architectures (386, amd64, arm64) and we likewise have 3 MSIs, but this change only has two MSIs listed.

Reply:
There's no third set of options for Install-ChocolateyPackage to specifically support arm64. Without getting bespoke, Chocolatey will give arm64 users the 32-bit version that'll work on their platform.

Reply:
Alright, have another look. That ought to work, although I don't have an arm64 device to check with.

Reply:
Thanks!

Reply:
I don't know how to test, either :)

/cc @dblohm7 @sailorfrag

Reply:
I have an arm64 device I can test on but I'll have to investigate how to get chocolatey to use this PR unless you have a link to instructions.

Reply:
@sailorfrag I think this is what you're looking for: https://community.chocolatey.org/courses/creating-chocolatey-packages/building-testing-and-pushing

Reply:
I'm preparing a VM as recommended in the instructions, it's taking a bit since this is the first time I've set up HyperV on that device. I expect to be able to try this out tomorrow.

Reply:
It's installing amd64 on the arm64 device 🤦🏻‍♂️ 

I'll see if I can figure out how to fix

Reply:
I think this is not trivial, chocolatey doesn't seem to have a general concept of arm64 native packages. I'd recommend dropping references to ARM64 for now and solving it in another PR.

Reply:
I'm still digging into this, here's the environment while running in chocolatey (which does not match what I get if I run powershell on its own)
```
Name                           Value
----                           -----
__COMPAT_LAYER                 DetectorsAppHealth Installer
ALLUSERSPROFILE                C:\ProgramData
APPDATA                        C:\Users\redacted\AppData\Roaming
CHOCOLATEY_VERSION             1.2.1.0
CHOCOLATEY_VERSION_PRODUCT     1.2.1
ChocolateyAllowEmptyChecksu... true
ChocolateyEnvironmentDebug     true
ChocolateyEnvironmentVerbose   true
ChocolateyInstall              C:\ProgramData\chocolatey
ChocolateyLastPathUpdate       133174216570752868
chocolateyPackageFolder        C:\ProgramData\chocolatey\lib\tailscale
chocolateyPackageName          tailscale
chocolateyPackageTitle         Tailscale
chocolateyPackageVersion       1.34.2
chocolateyPackageVersionPac... 0
ChocolateyPowerShellHost       true
chocolateyRequestTimeout       30000
chocolateyResponseTimeout      2700000
CommonProgramFiles             C:\Program Files\Common Files
CommonProgramFiles(Arm)        C:\Program Files (Arm)\Common Files
CommonProgramFiles(x86)        C:\Program Files (x86)\Common Files
CommonProgramW6432             C:\Program Files\Common Files
COMPUTERNAME                   redacted
ComSpec                        C:\WINDOWS\system32\cmd.exe
DriverData                     C:\Windows\System32\Drivers\DriverData
GIT_INSTALL_ROOT               C:\Users\redacted\scoop\apps\git\current
HOMEDRIVE                      C:
HOMEPATH                       \Users\redacted
IS_ADMIN                       true
IS_PROCESSELEVATED             true
IS_REMOTE                      true
IS_REMOTEDESKTOP               false
IS_SYSTEM                      false
LOCALAPPDATA                   C:\Users\redacted\AppData\Local
LOGONSERVER                    \\redacted
NUMBER_OF_PROCESSORS           8
OneDrive                       C:\Users\redacted\OneDrive
OneDriveConsumer               C:\Users\redacted\OneDrive
OS                             Windows_NT
OS_IS64BIT                     true
OS_NAME                        Windows 10
OS_PLATFORM                    Windows
OS_VERSION                     10.0.22621.0
packageFolder                  C:\ProgramData\chocolatey\lib\tailscale
packageName                    tailscale
packageTitle                   Tailscale
packageVersion                 1.34.2
Path                           C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\WINDOWS\Sy...
PATHEXT                        .COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC;.CPL
PROCESS_IS64BIT                true
PROCESSOR_ARCHITECTURE         AMD64
PROCESSOR_IDENTIFIER           ARMv8 (64-bit) Family 8 Model 805 Revision D0E, Qualcomm Technologies Inc
PROCESSOR_LEVEL                2053
PROCESSOR_REVISION             0d0e
ProgramData                    C:\ProgramData
ProgramFiles                   C:\Program Files
ProgramFiles(Arm)              C:\Program Files (Arm)
ProgramFiles(x86)              C:\Program Files (x86)
ProgramW6432                   C:\Program Files
PSExecutionPolicyPreference    Bypass
PSModulePath                   C:\Users\redacted\Documents\WindowsPowerShell\Modules;;;C:\Program Files\WindowsPowerShell\Modules;C:\WINDOWS\system...
PUBLIC                         C:\Users\Public
SystemDrive                    C:
SystemRoot                     C:\WINDOWS
TEMP                           C:\Users\redacted\AppData\Local\Temp\chocolatey
TMP                            C:\Users\redacted\AppData\Local\Temp\chocolatey
USER_DOMAIN                    redacted
USER_NAME                      redacted
USERDOMAIN                     redacted
USERDOMAIN_ROAMINGPROFILE      redacted
USERNAME                       redacted
USERPROFILE                    C:\Users\redacted
windir                         C:\WINDOWS
WSLENV                         WT_SESSION::WT_PROFILE_ID
WT_PROFILE_ID                  {redacted uuid}
WT_SESSION                     redacted uuid
```
`PROCESSOR_IDENTIFIER` seems to be the only thing I can suggest matching on, but that's kind of messy.